pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -87,6 +87,10 @@ def __init__(self,
         # indicate that we're ready
         self._write_packet_to_processor('s')
 
+        # log deprecation warning
+        if sys.version_info[:2] == (2, 7):
```

Comment:
I would use if major is 2 (wider condition) - just to be sure, since people can bring in their base images.